### ISSUE TITLE
Bot gets stuck in water if it tries to place a block under itself while Jumping/Swimming

### ISSUE THREAD

**Issue by IceTank** [PrismarineJS/mineflayer-pathfinder#54]
While in water the bot can get stuck if it tries to place a block.
This can happen when the bot is standing in 1 high water and tries to place a block to get up 1 block. In this case the bot tries to place a block under itself while jumping. While not in water the bot is able to jump high enough to place a block under itself but this is not the case in Water. In Water it is impossible to jump higher then the water Block the player or bot is in.  The Bot tries anyway and gets stuck.

Visualisation:
w = Water
o = Water Block the bot is trying to replace with a block while jumping and bot feet position
i = Bot head position  
s = Solid block
```  
  i
wwoww
sssss
```

**Comment by IceTank** [comment]
Posible fix:
Add this line to movements.js after line 251:
```
if (this.getBlock(node, 0, 0, 0).liquid) return
```

**Comment by SinanAkkoyun** [comment]
It still does not work for me!

**Comment by Karang** [comment]
Fixed, update to 1.3.0.
